src/endless/GeneralCode.cpp

```diff
@@ -7,4 +7,4 @@ void _print_hresult(const char *file, int line, const char *msg, HRESULT hr)
     LPCTSTR errMsg = err.ErrorMessage();
 
     uprintf("%s:%d %s (HRESULT: 0x%x %ls)", file, line, msg, hr, errMsg);
-}
```

Comment:
Super-annoying: when I edit in Vim, it adds the trailing newline; when I edit in VS, it removes it…